livedata/src/main/java/io/getstream/chat/android/livedata/ChatDomain.kt

```diff
@@ -148,6 +157,15 @@ interface ChatDomain {
                 }
             }
         }
+
+        private fun storeNotificationConfig(notificationConfig: NotificationConfig) {
+            if (NotificationConfigUnavailable != notificationConfig) {
+                syncModule.notificationConfigStore.apply {
+                    clear()
```

Comment:
Is it needed to "clear" before "put" a new value?
`put()` methods normally override previous value